terraform/environments/cooker/providers.tf

```diff
@@ -1,10 +1,14 @@
 ######################### Run Terraform via CICD ##################################
 # AWS provider for the workspace you're working in (every resource will default to using this, unless otherwise specified)
+# Also include the default_tags we want to use on systems
 provider "aws" {
   region = "eu-west-2"
   assume_role {
     role_arn = "arn:aws:iam::${local.environment_management.account_ids[terraform.workspace]}:role/MemberInfrastructureAccess"
   }
+  default_tags {
```

Comment:
This change is the only one you need

Reply:
Just cleared out the # local.tags that was causing the issue with 20+ changes appearing even though nothing had changed. Need to repeat this on other environments. I'll start with Sprinkler to confirm it's OK - probably new PRs for each

Reply:
@SteveLinden could you tidy up this PR first please?  It should only have one change in it - 
```
default_tags {
    tags = local.tags
 }
```

That is all that is needed, thanks!